widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -329,6 +329,10 @@ class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defSty
         controller?.navigateToCallSuccess()
     }
 
+    fun shouldShow(): Boolean {
+        return controller?.let { !it.isCallVisualizerOngoing() } ?: false
```

Comment:
This is a bit complicated topic to discuss async. But if shortly:
> we should abort the whole operation with an error(error dialog that chat is currently unavailable)

Agree, but for now, Henrik proposed this solution because it is simple and similar to some other scenarios. We will discuss in detail further improvements in Slack.
> or just an error log

Agree with that ➕ 

> I don't believe that mandatory components should ever be nullable

But as far as I understand this is an expected case. Call Visualizer has no chat so if the integrator for some reason forcefully started new intent with ChatACtivity it would miss crucial arguments/settings because it should not be opened during a Call VIsualizer engagement. Further improvements will be done later based on slack discussion

Reply:
@Pelkar @gugalo Added error log. Please have a look 7251b0ca85b15221a008a4cac7ff93131d78fb41